framework/source/class/qx/bom/client/EcmaScript.js

```diff
@@ -117,6 +117,26 @@ qx.Bootstrap.define("qx.bom.client.EcmaScript",
 
 
     /**
+     * Checks if 'find' is supported on the Array object.
+     * @internal
+     * @return {Boolean} <code>true</code>, if the method is available.
+     */
+    getArrayFind : function() {
```

Comment:
Based on the "intuitive return value function names" discussion we had recently, I'd suggest that ascertaining if something exists or is available should include one of those words in the name. I'd suggest *arrayFindAvailable* or *arrayFindExists*. That way, a return value of true clearly indicates that it is available or does exist. (I like the *available* form better.)